src/Assets/Bugsnag/Editor/BugsnagAddScriptingSymbol.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEditor;
+using UnityEngine;
+[InitializeOnLoad]
+public class BugsnagAddScriptingSymbol : MonoBehaviour
+{
+    private const string DEFINE_SYMBOL = "BUGSNAG_UNITY_WEB_REQUEST";
+
+    private static BuildTargetGroup[] _supportedPlatforms = { BuildTargetGroup.Android, BuildTargetGroup.Standalone, BuildTargetGroup.iOS, BuildTargetGroup.WebGL};
+
+    static BugsnagAddScriptingSymbol()
+    {
+        foreach (var target in _supportedPlatforms)
+        {
+            try
+            {
+                SetScriptingSymbol(target);
+            }
+            catch
+            {
+                // Some users might not have a platform installed, in that case ignore the error
+            }
+        }
+    }
+
+    static void SetScriptingSymbol(BuildTargetGroup buildTargetGroup)
+    {
+        var existingSymbols = PlayerSettings.GetScriptingDefineSymbolsForGroup(buildTargetGroup);
```

Comment:
Is this guaranteed to be non-null? `existingSymbols` is used without a check below and will prepend a semi-colon if it's empty.